packages/quantic/force-app/main/default/lwc/quanticQueryError/quanticQueryError.css

```
@@ -0,0 +1,20 @@
+.slds-error__box {
+    text-align: left;
+    background: var(--lwc-colorBackgroundAlt, #FFFF);
+    color: var(--lwc-colorTextPlaceholder, #706e6b);
+    font-style: normal;
+    font-weight: normal;
+    font-size: 14px;
+    line-height: 20px;
+    border: var(--lwc-borderWidthThin, 1px) solid var(--lwc-colorBorder, #dddbda);
```

Why not this instead
```suggestion
    border: var(--lwc-borderWidthThin, thin) solid var(--lwc-colorBorder, #dddbda);
```